index.js

```diff
@@ -4,6 +4,11 @@ module.exports = {
   },
 
   responseObject: function(response) {
+    try {
+      JSON.parse(response.xhr.responseText);
+    } catch(err) {
+      return {invalid_JSON: response.xhr.responseText};
```

Comment:
I think part of the problem right now is that we are getting an error here. It goes straight to trying to log the error in raven vs letting the consumer handle it. So far I was only looking at this in templategallery. This is where the error breaks https://github.com/yola/templategallery/blob/master/src/js/template_gallery/modals/save-site.js#L39, it doesn't get past this declaration and just tries to log the error with raven. If we return an object with the key `invalid_JSON`, the consumer can check for that if needed. But I might be missing a larger picture since I'm only looking at this one example...